swagger.yaml

```diff
@@ -9206,3 +9257,13 @@ definitions:
       cluster_id:
         type: string
         format: uuid
+
+  infra-env-image-url:
+    type: object
+    properties:
+      url:
+        type: string
+        description: Pre-signed URL for downloading the infraEnv discovery image.
+      exp:
+        type: string
+        description: Expiration time for the URL token in Epoch time.
```

Comment:
```suggestion
      expires_at:
        type: string
        format: date-time
        x-go-custom-tag: gorm:"type:timestamp with time zone"
```

Reply:
:man_shrugging: This was originally made specifically to match the claim in the token, but I guess there's no reason for that to be the case.

You're generally suggesting that this should be human readable rather than the epoch time?

Reply:
We always use date-time.  Any reason not to do that here?  And is this different from expires_at in the infra-env?

Reply:
This object is just for the API, do we still need the `gorm` tag? Does it do anything when the object is never saved directly to the db?